src/WebJobs.Script.Grpc/Abstractions/IWorkerConfigProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    /// <summary>
+    /// Enables a language worker implementor to specify how to create and configure the language worker process.
+    /// </summary>
+    public interface IWorkerConfigProvider
```

Comment:
Not sure this is truly a provider. This is really just handling and applying configuration, correct? Perhaps `IWorkerConfigurationHandler`?